### ISSUE TITLE
Map won't change.

### ISSUE THREAD

**Issue by M-archand** [TotallyMehis/Influx-Timer#27]
Using sm 1.8 build 6002 - influx bhop module 1.0.8.

Installed my server fresh with all server and gamemode cfgs set with times and other settings needed, map changes fine at the end (I use mapchooser).

As soon as I install Influx the map hangs at 0:00 forever and the mapchooser menu doesn't come up. Tried with influx_fixtimelimit.smx both enabled and disabled. Didn't change any server settings, which worked just minutes before enabling Influx.

Nothing in error logs.

**Comment by TotallyMehis** [comment]
What is your mp_timelimit? I haven't been able to reproduce the problem.

**Comment by M-archand** [comment]
Have tried mp_timelimit with 5,6,7,10,20,25,30 minutes, nothing seems to work

**Comment by TotallyMehis** [comment]
Using the timeleft command gives out the correct time left? Influx doesn't change anything regarding the time. Mapchooser should still start a vote couple minutes before the time limit.

**Comment by M-archand** [comment]
timeleft gives "[SM] Map will change after one more round" and mp_maxrounds is set to 1

**Comment by TotallyMehis** [comment]
That should mean the mp_timelimit isn't being set properly. You should see the time left if mp_timelimit =/= 0. Set mp_maxrounds to 0.

**Comment by M-archand** [comment]
mp_maxrounds is set to 1, when I set it to 0 the map ends as soon as it loads in and switch to a random map

**Comment by M-archand** [comment]
I don't really think my gamemode cvars are the issue. The core settings are pretty much the same on all my servers, I use mapchooser on all of them and have no issues. 

This issue started occurring after a sourcemod update a few months ago (I eventually gave up on the server because of it) but want to try and fix it again. I feel like it must be something to do with mapchooser because it simply doesn't appear at all now. Like I said, I installed the server with only the base sourcemod plugins enabled, and am using the same server and gamemode cfg's, in this scenario map chooser worked as intended, appeared with a couple minutes left, and the map changed at the end. As soon as I enabled all the Influx plugins mapchooser vote doesn't appear anymore.
